tests/LaunchDarkly.ClientSdk.Device.Tests/LaunchDarkly.ClientSdk.Device.Tests.csproj

```diff
@@ -0,0 +1,78 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net7.0-android; net7.0-ios</TargetFrameworks>

```

Comment:
For reviewers: This is the test app for Android and iOS now.  The platform specific tests are in the Platform folder.